docs/standard/serialization/binary-serialization.md

```diff
@@ -82,7 +82,7 @@ As the nature of binary serialization allows the modification of private members
 - <xref:System.Collections.Specialized.StringCollection?displayProperty=nameWithType>
 - <xref:System.Collections.Specialized.StringDictionary?displayProperty=nameWithType>
 - <xref:System.Collections.Stack?displayProperty=nameWithType>
-- `System.Collections.Generic.NonRandomizedStringEqualityComparer` (available in .NET Core 2.0.4 and later versions)
+- <xref:System.Collections.Generic.NonRandomizedStringEqualityComparer?displayProperty=nameWithType> (available in .NET Core 2.0.4 and later versions)
```

Comment:
I think this xref would generate the following link <https://docs.microsoft.com/en-us/dotnet/api/System.Collections.Generic.NonRandomizedStringEqualityComparer> which gives 404 not found.
/cc @mairaw